user/cron-jobs.md

```diff
@@ -27,6 +27,10 @@ Confirm that the cron job is displayed in your settings tab:
 
 ![cron job created](/images/cron-created.png "cron job created")
 
+## Skipping Cron Jobs
+
+Currently, a new cron job looks to the most recent commit for whether [`[ci skip]` or `[skip ci]`](/user/customizing-the-build/#Skipping-a-build) is included in the commit message. If the commit message contains at least one of these, the cron job will not be run. 
```

Comment:
Maybe we could specify a bit more the beginning with something similar to `a new cron job fetches the latest commit for the branch it's been set for` and then explain that if `[skip ci]` is in the message... 

What do you think?